onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/ONPRC_RestrictedIssueTest.java

```diff
@@ -0,0 +1,269 @@
+    }
+
+    public void doInit()
+    {
+        _containerHelper.createProject(getProjectName(), null);
+
+        // Create test users
+        USER1.create(this).addPermission("Editor", getProjectName());
+        USER2.create(this).addPermission("Editor", getProjectName());
+        ISSUE_CREATOR.create(this).addPermission("Editor", getProjectName());
+        FOLDER_ADMIN.create(this).addPermission("Folder Administrator", getProjectName());
+
+        // Add issue list definitions
+        _issuesHelper.createNewIssuesList(RESTRICTED_ISSUES_LIST, _containerHelper, true, false, false);
+        waitAndClickAndWait(Locator.linkContainingText(RESTRICTED_ISSUES_LIST));
+        _issuesHelper.goToAdmin();
+        _issuesHelper.setRestrictedIssueList(true);
+        _issuesHelper.setIssueAssignmentList("Site: Users");
+        clickButton("Save");
+
+        clickProject(getProjectName());
+        _issuesHelper.createNewIssuesList(UNRESTRICTED_ISSUES_LIST, _containerHelper, false, false, false);
+        waitAndClickAndWait(Locator.linkContainingText(UNRESTRICTED_ISSUES_LIST));
+        _issuesHelper.goToAdmin();
+        _issuesHelper.setIssueAssignmentList("Site: Users");
+        clickButton("Save");
+    }
+
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        _userHelper.deleteUsers(false, USER1, USER2, ISSUE_CREATOR, FOLDER_ADMIN);
+        _containerHelper.deleteProject(getProjectName(), afterTest);
+    }
+
+    @Override
+    public List<String> getAssociatedModules()
+    {
+        return Arrays.asList("ehr", "onprc_ehr");
+    }
+
+    @Override
+    protected String getProjectName()
+    {
+        return "RestrictedIssuesVerifyProject";
+    }
+
+    @Test
+    public void restrictedIssueTest()
+    {
+        clickProject(getProjectName());
+
+        // create a few issues in the restricted list
+        impersonate(ISSUE_CREATOR.getEmail());
+        clickAndWait(Locator.linkContainingText(RESTRICTED_ISSUES_LIST));
+        DetailsPage detailsPage = _issuesHelper.addIssue(String.format("Restricted issue assigned to (%s)", USER1.getUserDisplayName()), USER1.getUserDisplayName());
+        final String ISSUE_1 = detailsPage.getIssueId();
+        InsertPage insertPage = detailsPage.clickCreateNewIssue();
+        insertPage.title().set(String.format("Restricted issue assigned to (%s)", USER2.getUserDisplayName()));
+        insertPage.assignedTo().set(USER2.getUserDisplayName());
+        insertPage.save();
+        final String ISSUE_2 = insertPage.getIssueId();
+        stopImpersonating();
+
+        // verify site admins can see both issues (but not folder admins)
+        clickProject(getProjectName());
+        clickAndWait(Locator.linkContainingText(RESTRICTED_ISSUES_LIST));
+        verifyIssueAccess(ISSUE_1, true);
+        verifyIssueAccess(ISSUE_2, true);
+
+        impersonate(FOLDER_ADMIN.getEmail());
+        clickProject(getProjectName());
```

Comment:
Shouldnt it be ?
gotoProjectHome();
impersonate(FOLDER_ADMIN.getEmail());
waitAndClickAndWait(Locator.linkContainingText(RESTRICTED_ISSUES_LIST));

Reply:
Does it matter? Either should be starting from the project begin URL?